### PR TITLE
Fix loading spinner position

### PR DESCRIPTION
**Pull request by driesd**
### Description

This PR fixes the `loading spinner position` for our `Button` component.

#### Screenshot before this PR
Screenshot 2019-07-18 13 47 31

#### Screenshot after this PR
Screenshot 2019-07-18 13 46 31

### Breaking changes

None

### CODE REVIEW

File: src/components/button/theme.css [hunk 1]
```diff
@@ -502,7 +502,7 @@
 
 .spinner {
   left: 50%;
-  position: absolute;
+  position: absolute !important;
```
Comment (rathesDot) [inline src/components/button/theme.css]: Is this also an overwrite for another repository?

Comment (ArnaudWeyts) [reply src/components/button/theme.css]: Looks like it 😕

Comment (ArnaudWeyts) [reply src/components/button/theme.css]: I propose we merge this for now as it is actually broken in core, but we add a `@TODO` comment to atleast know it needs to be fixed. I looked into it for a bit but I couldn't find the cause.

Comment (rathesDot) [reply src/components/button/theme.css]: Thanks for the feedback! Let's get this merged for now!

**Pull request merged**